02_progress-steps/script.js

```diff
@@ -0,0 +1,80 @@
+const progress = document.getElementById('progress')
+const prev = document.getElementById('prev')
+const next = document.getElementById('next')
+const circles = document.querySelectorAll('.circle')
+
+//変数の値を宣言。1を代入。
+let currentActive = 1
+
+//.circleに.activeを付与剥奪する動作の宣言。
+const circleAddActive = function () {
+    //circlesのままだと配列ではないのでArray.prototype.slice.call(circles)で配列として呼び出す。
+    Array.prototype.slice.call(circles).map((circle, idx) => {
+        //対象の.classの配列のインデックス数値が変数"currentActive"よりも小さかったら、対象の.classにclass名activeを加えて、そうでなかったらactiveを取り除く
+        if(idx < currentActive) {
+            circle.classList.add('active')
+        } else {
+            circle.classList.remove('active')
+        }
+    })
+}
+
+//#progressのwidthの設定の宣言
+const progressWidth = function () {
+const actives = document.querySelectorAll('.active')
+//上で宣言した定数"progress"のwidthに、(.activeの数-1)÷(.circleの数-1)×100[単位：%]の値を代入。※activeの値はクリックすると変動する。※-1しているのは.progressのwidthの0からとするため(.activeは初めから付与されているものが1つある)
+progress.style.width = (actives.length - 1) / (circles.length - 1) * 100 + '%'
+}
+
+//#prevと#nextのアクティブ化の条件式の宣言
+const btnDisabled = function () {
```

Comment:
これは、可読性の観点です！

実務では `boolean`を判定したい場合( ここでは、disabled が true か false みたいなこと)
```suggestion
const isDisabled = function () {
```
ってやる場合が多いです！

`btnDisabled`でもわかりやすくて Good !